src/common/wallet/wallet-config-helper.ts

```diff
@@ -0,0 +1,27 @@
+import { persistOptional } from '@common/storage';
+import { WalletConfig } from '@stacks/wallet-sdk';
+import { createWalletGaiaConfig, getOrCreateWalletConfig, Wallet } from '@stacks/wallet-sdk';
+import { gaiaUrl } from '@common/constants';
+import { logger } from '@common/logger';
+
+export function saveWalletConfigLocally(walletConfig: WalletConfig) {
+  persistOptional('walletConfig', JSON.stringify(walletConfig));
+  return walletConfig;
+}
+
+export async function getWalletConfig(wallet: Wallet | undefined) {
+  if (!wallet) return;
```

Comment:
Can the logic for when it's undefined be handled by the code invoking this function?